Winleafs.Wpf/Views/Layout/CreateEffectWindow.xaml

```diff
@@ -0,0 +1,95 @@
+﻿<Window x:Class="Winleafs.Wpf.Views.Layout.CreateEffectWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:xctk="http://schemas.xceed.com/wpf/xaml/toolkit"
+        xmlns:local="clr-namespace:Winleafs.Wpf.Views.Layout"
+         xmlns:materialDesign="http://materialdesigninxaml.net/winfx/xaml/themes"
+        mc:Ignorable="d"
+        Title="{x:Static local:Resources.CreateEffectWindow}" Height="828" Width="1020" ResizeMode="CanMinimize">
```

Comment:
Have you tried to allow resizing of the window and how it would scale?

Reply:
This was a copy of the PercentageProfile window so I left it like this to be consistent. I wasn't sure if you wanted a `dialog` Window to be resizable as some people prefer them not to be. So happy to allow if you are?